sql/src/test/java/io/crate/analyze/symbol/format/SymbolPrinterTest.java

```diff
@@ -287,7 +287,7 @@ public void testMaxDepthFail() throws Exception {
         expectedException.expect(MaxDepthReachedException.class);
         expectedException.expectMessage("max depth of 5 reached while traversing symbol");
 
-        Symbol nestedFn = sqlExpressions.asSymbol("abs(sqrt(ln(1+1+1+1+current_timestamp(3)+1+1+1+1)))");
+        Symbol nestedFn = sqlExpressions.asSymbol("abs(sqrt(ln(1+1+1+1+(current_timestamp(3)::long)+1+1+1+1)))");
```

Comment:
Do we still need the cast here?

Reply:
Yes because `ln` doesn't accept timestamp as type, and timestamp would have the highest type precedence in this example.